components/PostPreview.tsx

```diff
@@ -1,19 +1,18 @@
-import React, { useContext } from 'react';
-import striptags from 'striptags';
+import React from 'react';
 import cx from 'classnames';
-import type { DualPost } from '../lib/types';
+import type { Lang, Post } from '../lib/types';
 import { englishMonths, spanishMonths } from '../lib/dates';
-import { LanguageContext } from '../lib/LanguageContext';
 import TeachingBadge from './TeachingBadge';
 import { getExcerpt } from '../lib/helpers';
 
 interface Props {
-  post: DualPost;
+  post: Post<Lang>;
+  category: 'teaching' | 'post';
```

Comment:
i'm curious why this is getting passed in as a separate prop.  isn't it on the `Post` type too?